brands/amenity/bank.json

```diff
@@ -3022,8 +3004,39 @@
       "name": "Finansbank"
     }
   },
+  "amenity/bank|First Bank~(North and South Carolina)": {
+    "countryCodes": ["us"],
+    "matchNames": [
+      "1st bancorp",
+      "1st bank",
+      "first bancorp"
+    ],
+    "tags": {
+      "amenity": "bank",
+      "brand": "First Bank",
+      "brand:wikidata": "Q5452332",
+      "brand:wikipedia": "en:First Bancorp",
+      "name": "First Bank"
+    }
+  },
+  "amenity/bank|First Bank~(Puerto Rico)": {
+    "countryCodes": ["us"],
+    "matchNames": [
+      "1st bancorp",
+      "1st bank",
+      "first bancorp"
+    ],
+    "tags": {
+      "amenity": "bank",
+      "brand": "First Bank",
+      "brand:wikidata": "Q5452333",
+      "brand:wikipedia": "en:First BanCorp",
```

Comment:
Note the distinction between “[First Bancorp](https://en.wikipedia.org/wiki/First_Bancorp)” and “[First BanCorp](https://en.wikipedia.org/wiki/First_BanCorp)”. 🤕

Reply:
ouch

Reply:
another reason why wikidata qids are better for everything